app/viewModels/ViewMovement.scala

```diff
@@ -33,7 +33,7 @@ object ViewMovement {
           .toLowerCase,
         "mrn"     -> o.movementReferenceNumber,
         "status"  -> o.status,
-        "actions" -> Seq("history") // TODO: Move this from the view to here. This will be decided based on the message type
+        "actions" -> Seq.empty[String] // TODO: Move this from the view to here. This will be decided based on the message type. Create these as objects too.
```

Comment:
Does it not make sense to have removed this entirely?

Reply:
Yup that would work, though I thought it was nice to have a little reminder there as we are planning to bring this back in the not too distant future.